src/App.css

```diff
@@ -54,3 +54,15 @@
     top: 25px
 }
 
+.review-comment-count {
+    padding: 20px
+}
+
+#comment-author {
+    text-decoration: underline;
+    color: rgb(106, 106, 106)
+}
+
+#comment-body {
+    border-style: solid;
+}
```

Comment:
I know you said the css is in the works, but it would be good if there was a border or background colour that showed more clearly which votes belonged to which comment